entries/l/lucidchart.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Lucidchart": {
+    "domain": "lucidchart.com",
+    "contact": {
+      "twitter": "LucidSoftware",
```

```suggestion
      "twitter": "LucidHelp",
```
Support accounts are generally preferred as they tend to be monitored more.

---

thank you 👍